google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -594,7 +600,7 @@ def hmac_keys service_account_email: nil, project_id: nil,
         def signed_url bucket, path, method: nil, expires: nil,
                        content_type: nil, content_md5: nil, headers: nil,
                        issuer: nil, client_email: nil, signing_key: nil,
-                       private_key: nil, query: nil, version: nil
+                       private_key: nil, query: nil, version: nil, virtual_hosted_style: nil
```

Comment:
bucket_bound_hostname isn't exposed in signed_url methods.

Reply:
Yes, will add now.

Reply:
Added, but PTAL at the docs description and suggest replacement if you have it.